projects/experimental/components/compass/compass.component.ts

```diff
@@ -0,0 +1,21 @@
+import {ChangeDetectionStrategy, Component, Input} from '@angular/core';
+import {MODE_PROVIDER} from '@taiga-ui/core';
+
+@Component({
+    selector: 'tui-compass',
+    template: '<div class="t-gradient"></div>',
```

Comment:
Nesting is unnecessary:
https://codepen.io/waterplea/pen/zYyROQN
I used CSS variable for rotation so the gradient tint does not rotate.

Reply:
fixed